src/buckets/actions/thunks.test.ts

```diff
@@ -0,0 +1,104 @@
+import {fetchAllBuckets, getBuckets} from './thunks'
+import * as api from 'src/client'
+import {fetchDemoDataBuckets} from 'src/cloud/apis/demodata'
+import {getMockAppState} from 'src/mockAppState'
+import {RemoteDataState} from '@influxdata/clockface'
+import {PublishNotificationAction} from 'src/shared/actions/notifications'
+
+jest.mock('src/client', () => ({
+  getBuckets: jest.fn(),
+}))
+
+jest.mock('src/cloud/apis/demodata', () => ({
+  fetchDemoDataBuckets: jest.fn(() => {
+    const res: ReturnType<typeof fetchDemoDataBuckets> = Promise.resolve([
+      {
+        id: 'demo-bucket',
+        retentionRules: [],
+        name: 'demo-buck',
+        readableRetention: 'retee',
+        type: 'demodata',
+      },
+    ])
+    return res
+  }),
+}))
+
+const mockGetBuckets = (shouldSucess: boolean) => {
+  const headers = {} as any
```

Comment:
shouldn't need to cast `as any` here. Could just be `const headers: any = {}`

Reply:
now casting whole mock function instead only headers